CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ include(SetBuildType)
 include(SetupPic)
 include(SetCxxStandard)
 include(StripSymbols)
+# We need Boost for InfoAtLink
+include(SetupBoost)
```

Comment:
Are you sure you didn't create this warning in the previous commit? I think it should be squashed into the previous commit.

Reply:
done